client/dist/styles/ratingsReviews.css

```diff
@@ -308,4 +345,36 @@
 
 .irt-hr {
   width: 100%;
+}
+
+/* STARS */
+
+.stars {
+  position: relative;
+  vertical-align: middle;
+  display: inline-block;
+  color: lightgray;
+  overflow: hidden;
+}
+
+.full-stars {
```

Comment:
Curious as to why stars class is relative position while full stars is absolute? Not to come at the CSS King, just wondering how relative and absolute positioning work together if screen size changes